seed-db/init.sh

```diff
@@ -2,5 +2,4 @@
 
 mongoimport --collection courses --drop --file fakeCourses.json --jsonArray --uri "mongodb://mongo:27017"
 mongoimport --collection regulations --drop --file fakeRegulations.json --jsonArray --uri "mongodb://mongo:27017"
-
-
+mongoimport --collection programmeDegrees --drop --file fakeProgrammeDegrees.json --jsonArray --uri "mongodb://mongo:27017"
```

Comment:
do we wanna stick with programmeDegrees or programmedegrees? The api currently requires programmedegrees

Reply:
I think programmeDegrees cause it's 2 words

Reply:
I originally went with programmedegrees cuz URIs dont have caps, but I dont have a strong preference, up to you guys